evm/src/lib.rs

```diff
@@ -34,10 +34,14 @@ pub mod vars;
 pub mod verifier;
 pub mod witness;
 
+use eth_trie_utils::partial_trie::HashedPartialTrie;
 // Set up Jemalloc
 #[cfg(not(target_env = "msvc"))]
 use jemallocator::Jemalloc;
 
 #[cfg(not(target_env = "msvc"))]
 #[global_allocator]
 static GLOBAL: Jemalloc = Jemalloc;
+
+pub type PartialTrie = HashedPartialTrie;
+pub type Node = eth_trie_utils::partial_trie::Node<PartialTrie>;
```

Comment:
Let me know how you feel about this. I think aliases make using these types a lot more readable, but I wasn't sure about placing this in `lib.rs`. It's needed by both `generation` and `cpu`, so `lib.rs` was the highest up parent module.